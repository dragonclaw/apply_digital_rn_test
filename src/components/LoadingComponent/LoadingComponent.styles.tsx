import styled from 'styled-components';
import {ActivityIndicator, View} from 'react-native';

export const LoadingComponentContainer = styled(View)`
  flex: 1;
`;
export const LoadingIndicator = styled(ActivityIndicator)`
  flex: 1;
  align-self: center;
  justify-content: center;
`;
