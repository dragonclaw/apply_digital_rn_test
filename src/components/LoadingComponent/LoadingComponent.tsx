import React from 'react';

import {
  LoadingComponentContainer,
  LoadingIndicator,
} from './LoadingComponent.styles';

const LoadingComponent = () => {
  return (
    <LoadingComponentContainer>
      <LoadingIndicator size={50} color={'#aaa'} />
    </LoadingComponentContainer>
  );
};

export default LoadingComponent;
