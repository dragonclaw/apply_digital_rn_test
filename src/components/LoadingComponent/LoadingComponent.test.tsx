import React from 'react';
import {render} from '@testing-library/react-native';
import LoadingComponent from './LoadingComponent'; // Adjust the import path as necessary

describe('LoadingComponent', () => {
  it('matches the snapshot', () => {
    const {toJSON} = render(<LoadingComponent />);
    expect(toJSON()).toMatchSnapshot();
  });
});
