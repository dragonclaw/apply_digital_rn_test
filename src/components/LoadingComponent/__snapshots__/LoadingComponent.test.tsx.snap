// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingComponent matches the snapshot 1`] = `
<View
  className="sc-braxZu ccBOXj"
>
  <ActivityIndicator
    className="sc-gJhJTp fSQPeg"
    color="#aaa"
    size={50}
  />
</View>
`;
